addons/s3/views.py

```diff
@@ -143,20 +143,15 @@ def create_bucket(auth, node_addon, **kwargs):
 
     try:
         utils.create_bucket(node_addon, bucket_name, bucket_location)
-    except exception.S3ResponseError as e:
+    except exceptions.NoCredentialsError as e:
         return {
-            'message': e.message,
+            'message': str(e),
```

Comment:
I am wondering if `str(e)` would reveal sensitive data? It depends on how `NoCredentialsError` implements it. Please double check.

Reply:
It won't, double-checked